README.md

```diff
@@ -1 +1,121 @@
+
+
+Each successful strike on a nail grants the player one drink to give out during the Drink phase. 
+Each miss is one drink the player must take themselves during the Drink phase.
+
+
+If a player hits the wrong nail it counts as a miss. If doing so results in the final center nail being hammered into the stump, A new center nail is created and the current player is removed from the game.
+
+
+#### Drink
+A "Drink" is a consistent unit which must be decided upon before the game begins.
+
+
+During the drink phase the player may give out drinks for each successful strike they made during the Strike phase. 
+For each strike made as a Lad, that player may give a drink out to another Lad. For each strike made as a Lord, that player may give a drink out to another Lord or make all remaining Lads drink at once.
+
+
+The final strike of a starting nail is considered a strike made as a Lord, as that player becomes a Lord simultaneously to striking the nail.
+
+
+The final strike of the game (the last center nail) is considered a strike made as a rank even higher than Lord. In this case all other players must drink.
+
+
+The player then must drink for each missed strike he or she performed during the Strike phase.
+
+
+Drinks may only be given out as long as the current player holds the hammer. If the player gives away the hammer without giving out drinks, their turn ends immediately and they forfeit giving out those drinks. However, they still must drink for each missed strike they made. Giving away the hammer for a hammer test does not count as forfeiting the hammer, and the player performing the hammer test must return the hammer when they are finished.
+
+
+At the end of the Drink phase, the game ends and the current player wins if all 3 center nails are hammered completely into the stump. Otherwise the player passes the hammer to the next player in turn order and they begin their turn on the Flip phase.
+
+
+### Hammer Test
+
+
+A Hammer Test can be performed by any player to determine if a nail has been completely hammered into the stump. For a nail that has been hit in vertically, the hammer is turned sideways the head is run over the nail from all angles. The nail is considered completely hammered in if the hammer does not clink with the nail. For a nail hit in sideways the same rule applies except the hammer is run over the nail both ways lengthwise. If everyone agrees a nail is completely hammered in then the Hammer Test can be skipped. Any player may challenge another player's nail during their turn via the Hammer Test.
+
+
+If the Hammer Test is performed during another players turn and that player gives up the hammer, it does not count as forfeiting the hammer; The hammer must be returned to that player after the test is complete.
+
+
+### Demotions
+
+
+If a Lord drops their hammer during the Flip phase they are instantly demoted to Lad. Before ending their turn they must drive in a new starting nail to the center position. 
```

Comment:
Yeah I agree center position definitely makes me think of the three in the middle.